runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -158,27 +159,73 @@ public constructor(
     @ImplicitReflectionSerializer
     public inline fun <reified T : Any> fromJson(tree: JsonElement): T = fromJson(context.getContextualOrDefault(T::class), tree)
 
-    companion object : StringFormat {
-        @Suppress("DEPRECATION")
+    /**
+     * The default instance of [Json] in the form of companion object.
+     */
+    @UnstableDefault
+    public companion object Default: StringFormat {
+        private const val message =
```

Comment:
The rationale behind `: StringFormat` is pretty simple -- it may be convenient to use it with some common methods.
But default instance is not exposed to the public deliberately: if one has to pass `Json` instance somewhere, they should create it manually